libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/stepper-advanced.service.ts

```diff
@@ -0,0 +1,392 @@
+import { Injectable } from '@angular/core';
+import { SdsFormlyTypes } from '@gsa-sam/sam-formly';
+import { FormlyFieldConfig } from '@ngx-formly/core';
+
+@Injectable()
+export class StepperAdvancedService {
+  getPermission(): FormlyFieldConfig {
+    return {
+      key: 'permissions',
+      fieldGroupClassName: 'grid-row',
+      fieldGroup: [
+        {
+          key: 'permissionsNeeded',
+          template: '<br/><h4><b>what permission</b></h4></br> Select the specfific <br>',
```

Comment:
What is this? If it's meant to show some sort of functionality can you update to be more applicable? Capitalize and have a call to action that makes sense.

Reply:
this just shows 'What permission' in the template, I capitalized it but didnt add anything else, because this is how it was originally when i copied it.

Reply:
I'd just take it out. It is an incomplete prompt that causes me to think that something didn't load rather than it being intentional.

Reply:
removed it.